File: CloudFlow/src/ro/cloudSoft/cloudDoc/presentation/client/shared/model/report/TaskuriCumulateReportFilterModel.java
package ro.cloudSoft.cloudDoc.presentation.client.shared.model.report;

import java.util.Date;

public class TaskuriCumulateReportFilterModel {

	private String zonaTask;
	private Date deLaData;
	private Date panaLaData;
	private Long userAsignat;
	private String status;

	public String getZonaTask() {
		return zonaTask;
	}

	public void setZonaTask(String zonaTask) {
		this.zonaTask = zonaTask;
	}

	public Date getDeLaData() {
		return deLaData;
	}

	public void setDeLaData(Date deLaData) {
		this.deLaData = deLaData;
	}

	public Date getPanaLaData() {
		return panaLaData;
	}

	public void setPanaLaData(Date panaLaData) {
		this.panaLaData = panaLaData;
	}

	public Long getUserAsignat() {
		return userAsignat;
	}

	public void setUserAsignat(Long userAsignat) {
		this.userAsignat = userAsignat;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

}

File: CloudFlow/FrontEnd-App/src/app/shared/model/reports/taskuri-cumulate-report-filter.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { JsonDateConverter } from "../../json-mapper";

@JsonObject
export class TaskuriCumulateReportFilterModel {

    @JsonProperty("zonaTask", String)
	public zonaTask: string = null;

	@JsonProperty("deLaData", JsonDateConverter)
	public deLaData: Date = null;

	@JsonProperty("panaLaData", JsonDateConverter)
    public panaLaData: Date = null;
    
    @JsonProperty("userAsignat", Number)
    public userAsignat: number = null;
    
    @JsonProperty("status", String)
    public status: string = null;
}